openwisp_radius/base/models.py

```diff
@@ -1007,7 +1007,13 @@ def get_or_create_user(self, row, users_list, password_length):
 
     def save_user(self, user):
         OrganizationUser = swapper.load_model('openwisp_users', 'OrganizationUser')
+        RegisteredUser = swapper.load_model('openwisp_radius', 'RegisteredUser')
         user.save()
+        registered_user = RegisteredUser(user=user)
+        registered_user.save()
```

Comment:
I suggest to move this line just before the `self.users.add(user)`, so you won't need the other save call below.